notebooks/multi_input_example.livemd

```diff
@@ -0,0 +1,83 @@
+# Multi Input example
+
```

Comment:
I think it might be a good idea to mention that XOR was a problem originally shown impossible for the simple perceptron to solve, but is an issue that deep learning overcomes